### PR TITLE
Ignore nonalphabet matches

### PR DESCRIPTION
**Pull request by Govind9**

### CODE REVIEW

**Comment by lintool** [comment]
1. We shouldn't bump up version unless we're planning on pushing out a new artifact. Are we?
2. Please add a test case to illustrate code changes.

**Comment by Govind9** [comment]
> 1. We shouldn't bump up version unless we're planning on pushing out a new artifact. Are we?
We need to change the version in order to resubmit to PyPi. Also, along with adding a rule for nonalphabetic matches, the number of lakes in the vocab has grown by almost 40% in the latest commit. Shouldn't this warrant a minor version change?
> 2. Please add a test case to illustrate code changes.
Added.

**Comment by lintool** [comment]
hi @Govind9 - now that we have an initial version published to PyPI already, I don't think we need to re-publish with every minor change...

**Comment by Govind9** [comment]
> hi @Govind9 - now that we have an initial version published to PyPI already, I don't think we need to re-publish with every minor change...

Reverted version back to 0.3.0

File: tests/test_waterwheel.py [hunk 1]
```diff
@@ -200,6 +200,7 @@ def test_issue20(self):
 
         for ent in doc.ents:
             self.assertIsNotNone(ent._.wikilink)
+    
     def test_province_abbrebiations(self):
```
Comment (lintool) [inline tests/test_waterwheel.py]: typo: abbreviations

File: waterwheel/waterwheel.py [hunk 2]
```diff
@@ -75,6 +75,9 @@ def __call__(self, doc: Doc):
             if any(t.ent_type for t in doc[start:end]) and not self.overwrite:
                 continue
             match_str = str(doc[start:end])
+            is_non_alhbetic = re.search('^[^a-zA-Z\d]+$', match_str) is not None
```
Comment (lintool) [inline waterwheel/waterwheel.py]: typo: alphabetic

**Comment by Govind9** [comment]
> Please fix typos.

Done.

**Pull request merged**